x/crisis/keeper/params.go

```diff
@@ -1,37 +0,0 @@
-package keeper
-
-import (
-	"context"
-
-	errorsmod "cosmossdk.io/errors"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/types/errors"
-	"github.com/cosmos/cosmos-sdk/x/crisis/types"
-)
-
-// GetConstantFee get's the constant fee from the store
-func (k *Keeper) GetConstantFee(ctx context.Context) (constantFee sdk.Coin, err error) {
```

Comment:
This is only used in Genesis export and getting the constant fee for invariance checks. Here the behavioural change is that before it wouldn't error if it did not exist, which is a bug since it's impossible to send invalid coins.